example/src/Utilities/payWithCard.js

```diff
@@ -0,0 +1,29 @@
+import { AdyenCSE, AdyenAction } from '@adyen/react-native';
+import { ENVIRONMENT } from '../Configuration';
+import ApiClient from './APIClient';
+import { checkoutConfiguration } from './AppContext';
+
+export async function payWithCard(unencryptedCard, configuration) {
+  const encryptedCard = await AdyenCSE.encryptCard(
+    unencryptedCard,
+    ENVIRONMENT.publicKey
+  );
+  const data = {
+    paymentMethod: {
+      type: 'scheme',
+      encryptedCardNumber: encryptedCard.number,
+      encryptedExpiryMonth: encryptedCard.expiryMonth,
+      encryptedExpiryYear: encryptedCard.expiryYear,
+      encryptedSecurityCode: encryptedCard.cvv,
+      threeDS2SdkVersion: AdyenAction.threeDS2SdkVersion,
+    },
+  };
+
+  var result = await ApiClient.payments(data, configuration, ENVIRONMENT.returnUrl);
```

Comment:
Same here we can use `let`.

Reply:
Fixed [97323d1](https://github.com/Adyen/adyen-react-native/pull/342/commits/97323d11bbb6a2b364c930e4265297ee7a598917)